source/initHQ/fortifyFOB.sqf

```diff
@@ -1,12 +1,17 @@
-_centerPos = _this select 3;
+_centerPos = (_this select 3) select 0;
 _actionID = _this select 2;
 _object = _this select 0;
+_fob = (_this select 3) select 1;
+aliveAllUnits = {alive _x} count allunits;
 
 if (commandpointsblu1 < 4) exitWith {hint "You don't have enough Command Points"};
 commandpointsblu1 = commandpointsblu1 - 4;
 publicvariable "commandpointsblu1";
-_object removeAction _actionID;
 
+
+if ((Warcom_Limiter_Param == 1) && (aliveAllUnits>99)) exitWith {hint "*DUWS AI Limiter is ENABLED!*\n\nTry Fortifying again when there are less than 100 AI units on the map"};
```

Comment:
Glad I could make sense this time, lol. It's just an old programmers trick. Try to have as little "magical numbers" as possible. In the long run it makes things easily changeable. If you only have one place to make an edit it's 5 minutes. If you have to make the same change in 30 different places it's no longer 5 minutes. :-)